### DISCUSSION TITLE
Error bar of the color same as a graph line

### DISCUSSION THREAD

**Discussion by nakanobutako** [sgratzl/chartjs-chart-error-bars#40] [Q&A]
![3C182B7E-9242-4D7A-B005-6B7416767E89](https://user-images.githubusercontent.com/116524823/197421120-cae83b51-b078-4ace-90c4-d76805197e5d.png)

How can you make the way with the color of a line and the error bar as the same color like a photograph?

**Comment by sgratzl** [comment]
have you tried setting one of the error bar styling options?

https://github.com/sgratzl/chartjs-chart-error-bars/blob/main/src/elements/render.ts#L17-L54
